### ISSUE TITLE
Android - Steps Screen - TalkBack Not Reading Items Under Topics Section as Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#34]
The items under the “Topics” section for each “Step” does not read them as links which they are because I knew that using them without TalkBack I can tap on them and they take me to the correct Topic screen.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. We can address this.
ZTI, can we please discuss on Monday? I want to ensure I know what we can do to rectify this issue.

Thanks,
Kyle

**Comment by KyleLivingstonDOL** [comment]
Hi John,

So this is similar #33 (which I think we've addressed and can close out). ILAB conducted many user interviews and gathered feedback. Some of the more pressing feedback came in the form of users telling us that they don't remember which items they've read and which they have not. As such, we implemented several things:
- A navigation bar that details what has been completed (green circle)
- Dynamic check boxes (blank boxes and check marks) to denote which topic might still need to be read
- Bookmarking

During UAT testing, ILAB is sufficiently pleased with how these items are working together and believe it (1) helps our user-based understand what they have and have not read, (2) address their concerns re: how they can reference material they've already read.

However, our contractors have told us that -- as of right now -- technology does not allow us to change the backend code to allow a talkback program to differentiate between the two "bullets" (i.e. have one be "unread" the other being "read".). Therefore, ILAB would like to propose the following:
1. Postpone this issue until after launch, at which point we can take a deeper look into how we might be able to use new technology and coding to allow us to have the talkback program differentiate between the two bullets
2. Update the app (Comply Chain will receive incremental updates) when the aforementioned technology is available.

Happy to chat further on this.

best,
Kyle

**Comment by jfc3-dol** [comment]
TalkBack is reading the items as a link and when double tapping on them going to the correct screen.

Closing issue.
